llvm/test/Transforms/LoopVectorize/X86/pr147398.ll

```diff
@@ -0,0 +1,126 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=loop-vectorize -force-vector-width=4 -S %s | FileCheck %s
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-linux-gnu"
+
+define i64 @test(ptr %arg2, i64 %dim) #0 {
```

Comment:
Could you add the test to `llvm/test/Transforms/LoopVectorize/X86/drop-poison-generating-flags.ll` which already has coverage for dropping poison generating flags?